locust_plugins/embedded_resource_manager.py

```diff
@@ -0,0 +1,179 @@
+from locust import HttpUser
+from locust.contrib.fasthttp import FastHttpUser
+from lxml import html, etree
+import logging
+import re
+
+
+class HttpUserWithResources(HttpUser):
+    """
+    provides embedded resource management for HttpUser
+    """
+
+    abstract = True
+
+    include_resources_by_default = True
+    default_resource_filter = ".*"
+    bundle_resource_stats = True
+    cache_resource_links = True
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        EmbeddedResourceManager(
```

Comment:
Would it  be possible to solve this using multiple inheritance instead? something like `class HttpUserWithResources(HttpUser, EmbeddedResourceManager):`

Passing all those variables looks a bit strange & un-necessary, if multiple inheritance is possible, you wouldnt need to pass them (and only need to list the default values in EmbeddedResourceManager)

Reply:
This is a good idea but I've struggled with getting a reference to the client methods. Not sure how much more time I want to put in to it at the moment.

Reply:
As long as it is internal to how the User works then we can fix it later.